README.md

```diff
@@ -328,7 +328,7 @@ python get_results.py
 
 ## 💻 LLM-generated Code
 
-We share pre-generated code samples from LLMs we have [evaluated](https://bigcodebench.github.io/leaderboard.html):
+We share pre-generated code samples from LLMs we have [evaluated](https://bigcode-bench.github.io/leaderboard.html):
```

Comment:
Thank you! @egor-bogomolov

Could you also help change this link to the HF leaderboard?